### PR TITLE
Clear-counters.py update

### PR DESCRIPTION
**Pull request by ksator**

### CODE REVIEW

**Comment by titom73** [comment]
Require PR #53 to be merged first and code update to reflect inventory API change

File: docs/requirements-and-installation.md [hunk 1]
```diff
@@ -65,16 +77,10 @@ Run these commands to verify:
...
 
-### Clone the repository and use `setup.py`
-
-Run these commands to clone the repository and to move to the new folder:
-
-```shell
-git clone https://github.com/arista-netdevops-community/network-test-automation.git
-cd network-test-automation
-```
+#### Using `python setup.py` commands
```
Comment (titom73) [inline docs/requirements-and-installation.md:83]: Seems some commands are missing ?

File: scripts/clear-counters.py [hunk 2]
```diff
@@ -23,35 +21,33 @@ def clear_counters(inventory, enable_pass):
...
-def report_uncleared_counters(inventory):
+def report_unreachable_devices(inventory):
     """
     report unreachable devices
     """
     devices = inventory.get_inventory(established_only = False)
     for device in devices:
         if device.established is False:
-            print("Could not clear counters on device " + str(device.host))
+            print("Could not connect on device " + str(device.host))
```
Comment (titom73) [inline scripts/clear-counters.py:44]: connect to

File: docs/requirements-and-installation.md (hunk 1 above)
Comment (ksator) [reply docs/requirements-and-installation.md:83]: I messed it up with my git commands. will repush what is missing

**Review by titom73 — APPROVED** [review]

**Pull request merged**